content/git/index.md

```diff
@@ -56,14 +57,41 @@ Better Example:
 
 * Use *dashes* to separate words.
 
-  [Merge](#merging) at will the personal branches to the team-wide branch
-  *after* rebasing onto it (in order to maintain a simple history). Eventually,
-  the team-wide branch will be merged to `master`.
+* When [merging](#merging) personal branches into shared branches, the personal
+  branch should be rebased onto the shared branch first in order to maintain a
+  clean history. If you need to clean up any commit messages or squash commits,
+  use the `-i` flag to enter the rebase interactive mode. When you are done with
+  the rebase, you will need to force push the new history to the remote.
+
+  ```shell
+  $ git checkout main && git pull
+  $ git checkout feat/personal-branch
```

Comment:
Up above, we have said that specific, descriptive branch names are best, and we used `feature/login_fix` as a bat example.  I wonder if we should make `feat/personal-branch` either _less_ specific to avoid making it _seem_ like an actual branch name: `git checkout my-personal-branch` or whether we should make it _more_ specific to agree with the rule above: `git checkout feature/update-nav-color` with a comment making it clear that this is on personal.

This is hair-splitting, and is not a blocking comment.